designs/virtualization/4.x/create-vm-dropdowns/create-vm-dropdowns.md

```diff
@@ -0,0 +1,112 @@
+'Operating system image source' section
+- 'Boot operating system via' (previously 'source')
+
+## General step
+
+![Create VM - General](img/1-1.jpg)
+Upon selecting an operating system, the configuration and Flavor (CPU and Memory) fields will be enabled and their defaults will be selected.
+
+![Create VM - General - VM configuration](img/1-1-1.jpg)
+VM Configuration options
+Server is the default
+
+![Create VM - General - flavor](img/1-1-2.jpg)
+Flavor options
+Medium is the default
+
+![Create VM - General](img/1-2-1.jpg)
+'Boot operating system via' field options
+
+![Create VM - General](img/1-2.jpg)
+Selecting an option from the 'Boot operating system via' field will expose below all the fields required to connect the VM to an OS image source.
+For example: on selecting the 'Existing PVC' option, additional fields will appear below, required to select a PVC available on the cluster.
+The disk name should be added here as well to establish a better link between this step and the Storage step.
+An info alert will notify the user that a disk will be added to to the disks list on the Storage step.
+
+![Create VM - OS disk on list](img/1-3.jpg)
+Within the storage step, the disk added on the general step will appear as the first item on the list.
+This disk cannot be deleted but can be modified.
+
+![Create VM - OS disk modal](img/1-3-1.jpg)
+Editing that disk will display a warning notification within the edit modal.
+
+## Netwroking step
+
+![Create VM - Networking -list](img/2-0.jpg)
+NIC list.
+For any VM source selection (with the exception of PXE) a default NIC will be added.
+This NIC exposes the VM to the cluster.
+
+![Create VM - Networking -Default](img/2-1.jpg)
+On editing this default NIC, an info alert will remind the user that modifying this NIC might break that 'exposing to cluster' configuration.
+
+![Create VM - Networking -Add](img/2-2.jpg)
+Add NIC modal.
+The Advanced drawer is closed by default, as seen in the previous image.
+All fields have moved into the advanced drawer, except the Name and Network fields.
+Network field does not require description for each option since these options are created and managed by the user.
+
+![Create VM - Networking -Model](img/2-3.jpg)
+'Model' dropdown options.
+ViritIO marked as (recommended)
+
+![Create VM - Networking -Binding](img/2-4.jpg)
+'Binding method' dropdown options (previously 'Type')
+
+## Storage
+
+![Create VM - Storage -list](img/3-0.jpg)
+Disks list.
+In case no disk type OS image source selected in the General step, a default disk will be created in order to make the VM operational.
+
+![Create VM - Storage -Default disk](img/3-0-0.jpg)
+Editing the default disk will present an info box notification.
+
+![Create VM - Storage -list](img/3-1.jpg)
+The Add/Edit modal has been modified.
+- Name
+- Type changed to 'Disk content'. This will change the configuration below
+- Advanced drawer added containing: 
+    - Interface
```

Comment:
IMO it is better to keep the Interface visibile since the value can vary according to the selected template and the user might be suprised what value was chosen by the UI. There is a quite complex logic which decides what the value will be according to the template

Reply:
Including @fabiand response from the doc:
> Because we have informations about what to set the default to (taken from the template)

Reply:
as I mentioned
- the user doesn't know what template was selected
- there might not be the right template for the user's use case
- the value can differ according to cdrom/disk and allowed/recommended buses